views.py

```diff
@@ -1,7 +1,20 @@
+from flask import render_template, flash, url_for, request, redirect
+from flask_security.utils import encrypt_password
+from flask_security import roles_accepted, roles_required
+from flask_admin import Admin, AdminIndexView
+from flask_login import login_required
 from app import app
+from models import *
+import os
```

Comment:
are these imports necessary for anything?

Reply:
Yes, views mainly interact with the models and database objects.

Reply:
but our views currently don't (since we have basically zero). We should only add imports as we need them, this avoids unnecessary clutter.